salt/modules/mac_timezone.py

```diff
@@ -0,0 +1,436 @@
+
+    :return: True if they are the same, False if not
+    :rtype: bool
+
+    CLI Example:
+
+    .. code-block:: bash
+
+        salt '*' timezone.zone_compare America/Boise
+    '''
+    current = get_zone()
+
+    if current != time_zone:
+        return False
+
+    return True
+
+
+def get_using_network_time():
+    '''
+    Display whether network time is on or off
+
+    :return: True if network time is on, False if off
+    :rtype: bool
+
+    CLI Example:
+
+    .. code-block:: bash
+
+        salt '*' timezone.get_using_network_time
+    '''
+    cmd = 'systemsetup -getusingnetworktime'
+    ret = _execute_return_result(cmd)
+
+    if _parse_return(ret) == 'On':
+        return True
+    else:
+        return False
+
+
+def set_using_network_time(enable):
+    '''
+    Set whether network time is on or off.
+
+    :param enable: True to enable, False to disable. Can also use 'on' or 'off'
+    :type: str bool
+
+    :return: True if successful, False if not
+    :rtype: bool
+
+    CLI Example:
+
+    .. code-block:: bash
+
+        salt '*' timezone.set_using_network_time True
+    '''
+    if enable not in ['On', 'on', 'Off', 'off', True, False]:
+        msg = 'Must pass a boolean value. Passed: {0}'.format(enable)
+        raise CommandExecutionError(msg)
+
+    if enable in ['On', 'on', True]:
+        enable = 'on'
+        expect = True
+    else:
+        enable = 'off'
+        expect = False
+    cmd = 'systemsetup -setusingnetworktime {0}'.format(enable)
+
+    _execute_return_success(cmd)
+
+    return expect == get_using_network_time()
+
+
+def get_time_server():
+    '''
+    Display the currently set network time server.
+
+    :return: the network time server
+    :rtype: str
+
+    CLI Example:
+
+    .. code-block:: bash
+
+        salt '*' timezone.get_time_server
+    '''
+    cmd = 'systemsetup -getnetworktimeserver'
+    ret = _execute_return_result(cmd)
+
+    return _parse_return(ret)
+
+
+def set_time_server(time_server):
+    '''
+    Designates a network time server. Enter the IP address or DNS name for the
+    network time server.
+
+    :param time_server: IP or DNS name of the network time server. 'Default'
+    will set it back to 'time.apple.com'
+    :type: str
+
+    :return: True if successful, False if not
+    :rtype: bool
+
+    CLI Example:
+
+    .. code-block:: bash
+
+        salt '*' timezone.set_time_server time.acme.com
+    '''
+    if time_server.lower() == 'default':
+        time_server = 'time.apple.com'
+    cmd = 'systemsetup -setnetworktimeserver {0}'.format(time_server)
+    _execute_return_success(cmd)
+
+    return time_server in get_time_server()
+
+
+def get_hwclock():
+    '''
+    Returns: Always returns 'UTC' because OS X Hardware clock is always UTC
+
+    CLI Example:
+
+    .. code-block:: bash
+
+        salt '*' timezone.get_hwclock
+    '''
+    return 'UTC'
+
+
+def set_hwclock():
+    '''
+    Setting not editable in Mac
+    '''
+    pass
```

Comment:
That's up to you. You could warn and then exit. Personally, I'd remove it. This just feels like dead code.